doc/man3/SSL_attach_stream.pod

```diff
@@ -69,6 +70,13 @@ stream. SSL_attach_stream() cannot be used if there is already a default stream
 associated with the QUIC connection SSL object; therefore, you may need to call
 SSL_detach_stream() first.
 
+If a stream is successfully attached to a QUIC connection SSL object using
+SSL_attach_stream(), the QUIC connection SSL object becomes responsible for
+managing its lifetime. Calling SSL_free() on the QUIC connection SSL object will
+free the stream automatically. Moreover, once the call to SSL_attach_stream()
+succeeds, the application must make no further use of the QUIC stream SSL object
+pointer that it passed to SSL_attach_stream().
```

Comment:
We should probably say that it is an error to attempt to attach a stream that has more than one reference to it.

Reply:
Unresolving this comment since I can't see a change that addresses it